cli/main.c

```diff
@@ -3,6 +3,8 @@
 #include <stdlib.h>
 #include <assert.h>
 
+#include <cerrno>
```

Comment:
There is `errno.h`, was the `CMakeLists.txt` building it as C++ by default? Or maybe some linter warning.